daemon/src/projection.rs

```diff
@@ -261,11 +265,17 @@ impl Cfd {
                 (CfdState::ContractSetup, vec![])
             }
             ContractSetupCompleted { dlc } => {
-                self.details.tx_url_list.push(TxUrl::new(
-                    dlc.lock.0.txid(),
+                // Lock transaction has shared output, but it likely has change
+                // output too that can be highlighted
+                let (lock_tx, lock_desc) = &dlc.lock;
+
+                let tx_url = TxUrl::from_transaction(
+                    lock_tx,
+                    &lock_desc.script_pubkey(),
                     network,
-                    TxLabel::Lock,
-                ));
+                    TxLabel::Collaborative,
```

Comment:
🔧 I think this should be `TxLabel::Lock` or similar.

Reply:
oops, of course! Don't know how this has slipt in.